store/postgres/src/entities.rs

```diff
@@ -360,37 +400,37 @@ impl<'a> Connection<'a> {
         // down, `migrating` remains set to `true` even though no work for
         // the migration is being done.
         Ok(
-            diesel::update(dsl::table.filter(dsl::subgraph.eq(subgraph.to_string())))
-                .set(dsl::migrating.eq(false))
-                .execute(&self.conn)
-                .map(|_| ())?,
+            diesel::update(
+                dsl::table.filter(dsl::subgraph.eq(self.storage.subgraph().to_string())),
+            )
+            .set(dsl::migrating.eq(false))
+            .execute(&self.conn)
+            .map(|_| ())?,
         )
     }
 
     pub(crate) fn find(
         &self,
-        subgraph: &SubgraphDeploymentId,
         entity: &String,
         id: &String,
         block: BlockNumber,
     ) -> Result<Option<Entity>, StoreError> {
-        match &*self.storage(subgraph)? {
+        match &*self.storage {
```

Comment:
`find` and `query`, if run against the metadata subgraph, would be running in a connection exclusive to it, correct?

Reply:
Yes, in that case `self.storage` and `self.metadata` are the same thing (that happens for example for `create_schema`) And I just checked that a query against the subgraph of subgraphs actually returns data.